### PR TITLE
Issue 91 add additional properties

### PR DESCRIPTION
**Pull request by quangkhoa**

### CODE REVIEW

File: src/parser/open-api/schemaParser.js [hunk 1]
```diff
@@ -116,17 +148,17 @@ export default function getUIReadySchema (jsonSchema) {
     return resolved.oneOf.map(
       (state) => {
         if (state.type === 'array') {
```
Comment (brendo) [inline src/parser/open-api/schemaParser.js]: This block is identical to the other block outside this condition, could they be extracted to a common method?

File: src/components/BodySchema/BodySchema.js [hunk 2]
```diff
@@ -31,11 +31,13 @@ export default class BodySchema extends Component {
     }
 
     const { expandedProps } = this.state
+    const hasNoAdditionalProperties = this.props.additionalProperties === false
 
     return (
       <table className={classNames(classes.bodySchema, classes[styleVariation], {
         [classes.hasTabs]: hasTabs
       })}>
+        {hasNoAdditionalProperties && <caption className='isAdditional'>These are the only possible properties:</caption>}
```
Comment (brendo) [inline src/components/BodySchema/BodySchema.js]: As you mentioned, need a little love with styling here (left align is a pretty good start, might not need much more!)

**Review by brendo — APPROVED** [review]

File: src/components/Property/Property.js [hunk 3]
```diff
@@ -38,6 +38,7 @@ export default class Property extends PureComponent {
     const {
       type, title, description, constraints, isRequired,
       defaultValue, onClick, isOpen, isLast,
+      attributes,
```
Comment (brendo) [inline src/components/Property/Property.js]: No need for new line, there's heaps of space above.

File: src/parser/open-api/schemaParser.js (hunk 1 above)
Comment (quangkhoa) [reply src/parser/open-api/schemaParser.js]: agreed. Extracted to a common method.

File: src/components/BodySchema/BodySchema.js (hunk 2 above)
Comment (quangkhoa) [reply src/components/BodySchema/BodySchema.js]: It's left aligned and italic now.

File: src/components/Property/Property.js (hunk 3 above)
Comment (quangkhoa) [reply src/components/Property/Property.js]: agreed. Moved up one line.

**Pull request merged**